docs/docs-content/clusters/pcg/deploy-app-pcg.md

```diff
@@ -625,13 +625,14 @@ your VMware vSphere environment.
 
 ## Wrap-Up
 
-In this tutorial, you learned how to deploy a VMware PCG using Palette CLI and utilize it to deploy a Palette cluster
-with a sample Kubernetes application to your VMware vSphere environment. Next, you accessed the Hello Universe
-application landing page through the exposed service URL.
+In this tutorial, you learned how to deploy a VMware PCG using Palette CLI. Then, you used the PCG to support the
+deployment of a Palette cluster with a sample Kubernetes application to your VMware vSphere environment. Next, you
```

Comment:
```suggestion
deployment of a Kubernetes cluster with a sample application to your VMware vSphere environment. Next, you
```

Removing the term "Palette cluster" as we are deploying a Kubernetes cluster. And the application is just a regular application :)